docs/20_samples/10_sample-edition/includes/titletab/README.md

```diff
@@ -0,0 +1,11 @@
+# Title tab 
+
+The title tab appears in all the views proposed by the `Open/edit sample` module. When you write something in this tab, it would be automatically updated in the report, but also in the `Title` tab of the other views. 
```

Comment:
```suggestion
The title tab appears in all the views proposed by the `Open/edit sample` module. When you write something in this tab, it will automatically be updated in the report, but also in the `Title` tab of the other views. 
```